.gitignore

```
@@ -2,4 +2,6 @@ __pycache__/
 **/*.mp4
 
 *.code-workspace
-.vscode/
\ No newline at end of file
+.vscode/
+timestamps.csv
```

```suggestion
**/*.csv
```
Better exclude all csv file, right?